src/components/interface/Transactions.vue

```diff
@@ -264,11 +275,9 @@ export default defineComponent({
 
     function sumTransfers(transfers: Transfer[]): string {
       let sum = new BigNumber(0);
-
       for (let i in transfers) {
-        if (transfers[i].amount?.isGreaterThan(new BigNumber(0))) sum = sum.plus(transfers[i].amount ?? new BigNumber(0));
+        if(transfers[i].amount > 0) sum = sum.plus(transfers[i].amount);
```

Comment:
<img width="679" alt="Screen Shot 2021-09-21 at 13 22 21" src="https://user-images.githubusercontent.com/22848332/134242049-c58f75c8-f69f-4e09-8a4f-07790fbc407d.png">

Since `transfers[i].amount` is of type `BigNumber`, the recommended way to compare is to use bignumber.js's api (👇🏻).

<img src="https://raw.githubusercontent.com/MikeMcl/bignumber.js/gh-pages/API.png" />

If you _really_ need to convert a BigNumber to compare with a number, then you can do this:

`+transfers[i].amount > 0` **or** `transfers[i].amount.toNumber() > 0` (both yield the same result; although unsure if at the cost of precision)

Reply:
Made recommended change to access BigNumber functions:

 `const amount = new BigNumber(transfer.amount ?? 0);
 if(amount.isGreaterThan(0)) sum = sum.plus(amount);`